lib/disability_compensation/requests/form526_request_body.rb

```diff
@@ -8,8 +8,8 @@ class Dates
     include ActiveModel::Serialization
     include Virtus.model
 
-    attribute :begin_date, Date
-    attribute :end_date, Date
+    attribute :begin_date, String
+    attribute :end_date, String
```

Comment:
Is this due to formatting? Double checking, is this not something we can enforce via `format`?

EDIT: I see the conversions in the transform and specs, but consider this might be something we want enforced form-side as well

Reply:
>  Is this due to formatting? Double checking, is this not something we can enforce via format?

yes.

and dates are already transmogrified in the data_all_claims method before they reach here so we were double transforming (which is what i think you were alluding to?)

Reply:
Gotcha, I'm more curious if we want to solution this to keep the attribute `Date` type. Let me look into `data_all_claims`.

Question might be better phrased as, do we need this to be a `String`? And why, if so?

Reply:
i chose to keep as a string because
1. it was already being sent as the format we need
2. ease of transforming the request body class into camel-cased-lower json: we don't have to shift through all of the data and transform it **_yet again_** when it gets to this phase. if we keep as a date, we need more transforming code when it actually gets sent to lighthouse

Reply:
Ah I see that now, thanks for the clarification. This may have to be revisited once we move to a single 3rd-party provider, but for our purposes it makes sense